src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -173,6 +173,29 @@ then you have to download the database driver and put it at: *PROACTIVE_HOME/add
 
 Scheduler, and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same database.
 
+[[_backup]]
+=== Configure automated backup
+
+ProActive Workflows & Scheduling features an automated backup mechanism which can be used to save the server state at regular intervals. Usage of this mechanism will depend on the database in use by the server:
+
+* In case the default database is used (HSQLDB), the backup mechanism can be used to store the database content, data and log files of the server.
```

Comment:
sorry, English fixing:
the backup mechanism can store the database contents, data ...